libs/components/forms/src/lib/modules/file-attachment/file-drop.component.spec.ts

```diff
@@ -474,7 +474,7 @@ describe('File drop component', () => {
 
     expect(typeError).toBeVisible();
     expect(typeError.textContent).toContain(
-      'Upload a file of type image/png, image/jpeg.',
+      'Upload a file of one of the following types: PNG, JPEG.',
```

Comment:
This message doesn't match the text in the resources string template?

Reply:
updated